documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -344,9 +337,9 @@ $ sudo raspi-config nonint do_hostname <hostname>
 ----
 
 [[boot-options-nonint]]
-==== Boot/auto login
+==== Boot/Auto login
 
-Select whether to boot to console or desktop, and whether or not your Raspberry Pi automatically logs into your current user account when powered on.
+Boot to console or desktop with the option of an automatic login to your current user account.
```

Comment:
```suggestion
Select the following behaviour at boot time:

* whether to boot to console or desktop
* whether your Raspberry Pi automatically logs into your current user account when powered on
```

Suggestion: I think your streamlining makes it less clear that automatic login is a separate option here, and introduces some ambiguity about whether you can use automatic login if you boot to console. I think an unordered list would help?

Maybe someday we can get the engineers to split this up into two separate options :-)